joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/dataflow/DataFlowTests.scala

```diff
@@ -889,7 +889,7 @@ class DataFlowTests extends RubyCode2CpgFixture(withPostProcessing = true, withD
     "find flows to the sink" in {
       val source = cpg.identifier.name("x").l
       val sink   = cpg.call.name("puts").l
-      sink.reachableByFlows(source).l.size shouldBe 2
+      sink.reachableByFlows(source).l.size shouldBe 1
```

Comment:
@DavidBakerEffendi Shouldn't the dataflow count be `2` as there are `2 identifiers` at line number `884` and `885`?

Reply:
For the statement `x = [1,2,3]` looks like we are not creating identifier, because I have observed this behaviour in other frontends as well

Reply:
@khemrajrathore Good catch, I'll follow up on that this morning

Reply:
Seems there are two identifiers being generated, but looking into the CFG and DDG now

Reply:
Update: The Java equivalent seems to be exactly the same, though I did find some properties incorrectly filled in for the `arrayInitializer` call node. I will test what the difference is before and after this diff.